autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -436,7 +436,7 @@ async def on_node_execution(
             graph_id=node_exec.graph_id,
             node_eid=node_exec.node_exec_id,
             node_id=node_exec.node_id,
-            block_name="-",
+            block_name=b.name if (b := get_block(node_exec.block_id)) else "-",
```

Comment:
Should we use something a bit more clear than -

Reply:
This is for logging, so totally fine, and it has been this way, I'm just improving it now